ocaml/xapi/helpers.ml

```diff
@@ -1234,3 +1234,122 @@ let host_supports_hvm ~__context host =
 let env_with_path env_vars =
   Array.of_list @@
   List.map (fun (k,v) -> Printf.sprintf "%s=%s" k v) (("PATH", String.concat ":" Forkhelpers.default_path)::env_vars)
+
+(** wrapper around the stunnel@xapi systemd service.
+  * there exist scripts (e.g. xe-toolstack-restart) which also manipulate
+    the stunnel daemon but they do this directly (not via ocaml). *)
+module Stunnel : sig
```

Comment:
It's not a problem, but I would have just put this into a new file.

Reply:
if the module were to get much bigger i would agree